ansible/docker.yml

```diff
@@ -30,13 +31,15 @@
       register: swarm_status
       changed_when: "'active' not in swarm_status.stdout_lines"
     - name: initialize swarm
-      shell: >
+      shell: |
+        docker swarm leave --force || echo "leaving swarm for multiple setups"
```

Comment:
Could you please clarify the intent of having the `docker swarm leave --force` here?

Reply:
Before I had `docker swarm leave --force` multiple setups would cause a docker failure. This was a workaround to prevent the error.